fixtures/pytest_store.py

```diff
@@ -207,3 +207,8 @@ def write_line(line, **kwargs):
         # resume capturing
         with diaper:
             store.capturemanager.resumecapture()
+
+
+def factory(cls, appliance_context=None, *args, **kwargs):
```

Comment:
i think we might instead want to use utils.appliance.get_or_create_current_appliance directly there